README.md

```diff
@@ -302,59 +302,57 @@ For more advanced use cases of forward references, you can use `plum.type.Promis
 
 ## Keyword Arguments and Default Values
 
-Keyword arguments can be used, but are *not* dispatched on.
-As a convention, arguments with default values should always be called as keyword 
-arguments.
-
-Example:
+Default arguments can be used. The type annotation must match the default value otherwise 
+an error is thrown. 
+Different default values can be used in different dispatch rules.
 
 ```python
 from plum import dispatch
 
 @dispatch
-def f(x, option="a"):
-    return f"Value for option: {option}"
+def f(x:int, y:int=3):
```

Comment:
```suggestion
def f(x: int, y: int = 3):
```

(I think this is what Black prescribes.)

Reply:
yeah, ideally black could be run on readme.md... 
I'm very bad at enforcing style manually